src/routes/PostDetail/PostDetail.js

```diff
@@ -164,11 +167,17 @@ export default class PostDetail extends Component {
         currentUser={currentUser}
       />
     )
-
+    MAX_THREAD_PREVIEW_LENGTH
     return (
       <ReactResizeDetector handleWidth handleHeight={false} onResize={this.handleSetComponentPositions}>
         {({ width, height }) => (
           <div styleName={cx('post', { noUser: !currentUser, headerPad: atHeader })}>
+            <Helmet>
+              <title>
+                {`Hylo: ${post.title || TextHelpers.truncateHTML(post.details, MAX_DETAILS_LENGTH)}`}
+              </title>
+              <meta name='description' content={TextHelpers.truncateHTML(post.details, MAX_DETAILS_LENGTH)} />
+            </Helmet>
```

Comment:
what about og:image and/or other image tags when the post has an image?

Reply:
OG tags help with previews, right? Do they have other purposes? Like are there accessibility meta tags? Because the OG ones won't be rendered unless we start doing SSR

https://stackoverflow.com/a/70796399/3288707